apps/challenges/views.py

```diff
@@ -302,6 +302,23 @@ def add_participant_team_to_challenge(
         .values_list("user", flat=True)
     )
 
+    # Check if all participants have verified emails
+    unverified_emails = []
+    for user in participant_team_user_ids:
+        try:
+            unverified_user = EmailAddress.objects.get(user=user, verified=False)
+            unverified_emails.append(unverified_user.email)
+        except EmailAddress.DoesNotExist:
+            continue
+
+    if unverified_emails:
+        message = "All team members need to have verified emails in order to participate. \
```

Comment:
@nikochiko rest of the changes LGTM but can we rephrase the message. It seems quite difficult to follow.

Reply:
@Ram81 Sure! How about, simply 'The following users have unverified emails:'?

Reply:
Or this is @KhalidRmb 's message : "Sorry, all team members must be verified through Email before participating.", 
then we can change to "Sorry, all team members must be verified through Email before participating. The following users have unverified emails: ..."

Reply:
Done!